README-zh.md

```diff
@@ -66,17 +66,17 @@ Spring Cloud 使用 Maven 来构建，最快的使用方式是将本项目 clone
 
 如果需要使用已发布的版本，在 `dependencyManagement` 中添加如下配置。
 ```xml
-	<dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>com.alibaba.cloud</groupId>
-                <artifactId>spring-cloud-alibaba-dependencies</artifactId>
-                <version>2.2.5.RELEASE</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.alibaba.cloud</groupId>
+            <artifactId>spring-cloud-alibaba-dependencies</artifactId>
+            <version>2.2.5.RELEASE</version>
```

Comment:
Change to the latest version 2.2.6.RELEASE

Reply:
The latest commit has updated this, please check.